drivers/hud_pit/spec/models/questions/projects.rb

```diff
@@ -0,0 +1,794 @@
+        expect(result).to eq(10)
+      end
+    end
+
+    describe 'Organization N - SO:' do
+      let(:row_number) { 46 }
+      it 'Project Name' do
+        result = cell_result(question: question, cell_name: column_names[:'Project Name'] + row_number.to_s)
+        expect(result).to eq('Organization N - SO')
+      end
+      it 'Client Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Client Count'] + row_number.to_s)
+        expect(result).to eq(1)
+      end
+      it 'Household Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Household Count'] + row_number.to_s)
+        expect(result).to eq(1)
+      end
+    end
+
+    describe 'Organization G - TH - 2:' do
+      let(:row_number) { 47 }
+      it 'Project Name' do
+        result = cell_result(question: question, cell_name: column_names[:'Project Name'] + row_number.to_s)
+        expect(result).to eq('Organization G - TH - 2')
+      end
+      it 'Client Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Client Count'] + row_number.to_s)
+        expect(result).to eq(2)
+      end
+      it 'Household Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Household Count'] + row_number.to_s)
+        expect(result).to eq(2)
+      end
+    end
+
+    describe 'Organization Q - ES - 2:' do
+      let(:row_number) { 48 }
+      it 'Project Name' do
+        result = cell_result(question: question, cell_name: column_names[:'Project Name'] + row_number.to_s)
+        expect(result).to eq('Organization Q - ES - 2')
+      end
+      it 'Client Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Client Count'] + row_number.to_s)
+        expect(result).to eq(1)
+      end
+      it 'Household Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Household Count'] + row_number.to_s)
+        expect(result).to eq(1)
+      end
+    end
+
+    describe 'Organization U - ES:' do
+      let(:row_number) { 49 }
+      it 'Project Name' do
+        result = cell_result(question: question, cell_name: column_names[:'Project Name'] + row_number.to_s)
+        expect(result).to eq('Organization U - ES')
+      end
+      it 'Client Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Client Count'] + row_number.to_s)
+        expect(result).to eq(9)
+      end
+      it 'Household Count' do
+        result = cell_result(question: question, cell_name: column_names[:'Household Count'] + row_number.to_s)
+        expect(result).to eq(6)
+      end
+    end
+  end
+end
+
+RSpec.configure do |rspec|
+  rspec.include_context 'adult and child', include_shared: true
+end
+
+def cell_result(question:, cell_name:)
+  report_result.answer(question: question, cell: cell_name).summary
+end
```

Comment:
This is totally fine to have in each of these, but I think this method is the same, maybe it can be DRY'ed up.